XPlat/Shopify/iOS/source/Shopify.iOS/Extras.cs

```diff
@@ -0,0 +1,231 @@
+﻿using System;
+using System.Runtime.InteropServices;
+using System.Threading;
+using System.Threading.Tasks;
+using Foundation;
+using ObjCRuntime;
+using PassKit;
+
+namespace Shopify.Buy
+{
+	//public static class BUYRuntime
+	//{
+	//	// extern NSSet * class_getBUYProperties (Class clazz) __attribute__((overloadable));
+	//	[DllImport ("__Internal")]
+	//	static extern NSSet class_getBUYProperties (Class clazz);
+	//}
+
+	partial class BuyClient
```

Comment:
Can you just use the [Async](https://developer.xamarin.com/guides/cross-platform/macios/binding/binding-types-reference/#AsyncAttribute) attribute in the ApiDefinition?